lib/common/include/scipp/common/span.h

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#pragma once
+
+#define TCB_SPAN_NAMESPACE_NAME scipp
```

Comment:
Not sure, it is generally strongly discouraged to add to `std`, or even undefined behavior? I don't know if backporting may be an exception, but since we do not know what this span implementation does internally...?